troupon/static/scss/partials/_components.scss

```diff
@@ -56,6 +56,10 @@ Buttons
   margin: 15px auto;
 }
 
+a.btn-action, a.btn-action-alt {

```

Comment:
Avoid qualifying class selectors with an element.<br>Each selector in a comma sequence should be on its own single line